modello-core/src/main/java/org/codehaus/modello/plugin/AbstractModelloGenerator.java

```diff
@@ -57,6 +58,14 @@
 public abstract class AbstractModelloGenerator implements ModelloGenerator {
     private final Logger logger = LoggerFactory.getLogger(getClass());
 
+    private static final Map<String, String> PLURAL_EXCEPTION = new HashMap<>();
+
+    static {
+        PLURAL_EXCEPTION.put("children", "child");
+        PLURAL_EXCEPTION.put("licenses", "license");
+        PLURAL_EXCEPTION.put("series", "series");
+    }
```

Comment:
I think there are a couple of additional exceptions: men -> man, women -> woman, mice -> mouse, geese -> goose, feet -> foot, teeth -> tooth, people -> person.